Makefile

```
@@ -204,3 +204,29 @@ migrate: migrate-lms migrate-cms
 # Part of https://github.com/openedx/wg-developer-experience/issues/136
 ubuntu-requirements: ## Install ubuntu 22.04 system packages needed for `pip install` to work on ubuntu.
 	sudo apt install libmysqlclient-dev libxmlsec1-dev
+
+eslint:
+	python scripts/quality_test/quality_test.py eslint
+
+stylelint:
+	python scripts/quality_test/quality_test.py stylelint
+
+xsslint:
+	python scripts/quality_test/quality_test.py xsslint
+
+pycodestyle:
+	pycodestyle .
+
+pi_check:
+	python scripts/quality_test/quality_test.py pii_check
+
+check_keyword:
+	python scripts/quality_test/quality_test.py check_keywords
+
+test-js:
+	python scripts/js_test.py --option jstest
+
+test-coverage:
+	python scripts/js_test.py --option coverage
+
+quality-test: pycodestyle eslint stylelint xsslint pi_check check_keyword
```

```suggestion
quality: pycodestyle eslint stylelint xsslint pii_check check_keywords
```

* Most repositories other than edx-platform have `make quality`, so let's use that target name here too.
* For completeness, we should have a `make pylint` target too that just runs `pylint`, and it should be part of `make quality`